docs/sources/reference/builder.md

```diff
@@ -626,8 +626,7 @@ ENTRYPOINT ["/usr/sbin/apache2ctl", "-D", "FOREGROUND"]
 
 If you need to write a starter script for a single executable, you can ensure that
 the final executable receives the Unix signals by using `exec` and `gosu`
-(see [the Dockerfile best practices](/articles/dockerfile_best-practices/#entrypoint)
-for more details):
```

Comment:
Because that link doesn't tell the reader anything about `exec` and `gosu` - and much of the content about using `ENTRYPOINT` isn't there, its in the builder doc.